scripts/netlify-switch.js

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env node
```

Comment:
`netlify.toml` is an optional file in the setup and you will most probably be able to achieve the same simply deleting it and passing arguments to the specific tools.

For instance, the build command is part of the npm script and you can call it directly from [drone's commands](https://github.com/ec-europa/ecl-twig/pull/313/files#diff-3216dfff0ed3e301453e6799e8c367e2R56) and the publishing directory is a flag in the [deploy command of the cli](https://cli.netlify.com/commands/deploy) so it's possible to pass it in [this command](https://github.com/ec-europa/ecl-twig/pull/313/files#diff-3216dfff0ed3e301453e6799e8c367e2R63).